Wikipedia/Code/AccountCreationViewController.swift

```diff
@@ -0,0 +1,285 @@
+
+import UIKit
+
+class AccountCreationViewController: UIViewController, CaptchaViewControllerRefresh, UITextFieldDelegate, UIScrollViewDelegate {
+    @IBOutlet var usernameField: UITextField!
+    @IBOutlet var passwordField: UITextField!
+    @IBOutlet var passwordRepeatField: UITextField!
+    @IBOutlet var emailField: UITextField!
+    @IBOutlet var captchaContainer: UIView!
+    @IBOutlet var scrollView: UIScrollView!
+    @IBOutlet var loginButton: UILabel!
+    @IBOutlet var titleLabel: UILabel!
+    @IBOutlet var usernameUnderlineHeight: NSLayoutConstraint!
+    @IBOutlet var passwordUnderlineHeight: NSLayoutConstraint!
+    @IBOutlet var passwordConfirmUnderlineHeight: NSLayoutConstraint!
+    @IBOutlet var emailUnderlineHeight: NSLayoutConstraint!
+    @IBOutlet var spaceBeneathCaptchaContainer: NSLayoutConstraint!
+    @IBOutlet var createAccountContainerView: UIView!
+
+    fileprivate var captchaId: NSString? = ""
+    fileprivate var rightButton: UIBarButtonItem?
+    public var funnel: CreateAccountFunnel?
+    fileprivate var captchaViewController: CaptchaViewController?
+    
+    func scrollViewDidScroll(_ scrollView: UIScrollView) {
+        // Disable horizontal scrolling.
+        scrollView.contentOffset = CGPoint(x: 0.0, y: scrollView.contentOffset.y)
```

Comment:
I don't see`alwaysBounceHorizontal` enabled here and it should be off by default. Is it enable d in the nib? The other key is ensuring the scrollView's `contentSize.width` is less than or equal to its `bounds.size.width`